cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -408,16 +409,31 @@ static Long time(HttpServletRequest request) {
     return timeString.toUpperCase().equals("NOW") ? System.currentTimeMillis() : Long.parseLong(timeString);
   }
 
-  static long startMs(HttpServletRequest request) {
-    String parameterString = caseSensitiveParameterName(request.getParameterMap(), START_MS_PARAM);
-    return parameterString == null ? DEFAULT_START_TIME_FOR_CLUSTER_MODEL
-                                   : Long.parseLong(request.getParameter(parameterString));
+  @Nullable
+  static Long startMs(HttpServletRequest request) {
+    return parseParamAsLong(request, START_MS_PARAM);
   }
 
-  static long endMs(HttpServletRequest request) {
-    String parameterString = caseSensitiveParameterName(request.getParameterMap(), END_MS_PARAM);
-    return parameterString == null ? System.currentTimeMillis()
-                                   : Long.parseLong(request.getParameter(parameterString));
+  @Nullable
+  static Long endMs(HttpServletRequest request) {
+    return parseParamAsLong(request, END_MS_PARAM);
+  }
```

Comment:
Having separate functions for `startMs` and `startMsOrDefault` (same issue for `end`) seems a little redundant (e.g. see code repetition for `parseParamAsLong(request, START_MS_PARAM)` between these functions). 

I wonder if we can make it more concise by dropping `endMs` and `startMs` functions. Their callers can call `startMsOrDefault` and `endMsOrDefault` with `null` as the default parameter.